### PR TITLE
Consolidated, rebased version of the remote organizer feature

### PR DESCRIPTION
**Pull request by ralflang**
Consolidated, rebased version of the remote organizer feature (minus CHANGES and package.xml because they are a moving target)

### CODE REVIEW

File: kronolith/templates/dynamic/edit.inc [hunk 1]
```diff
@@ -93,6 +93,16 @@
   </p>
 </div>
 
+<div id="kronolithEventAttend" style="display:none">
```
Comment (yunosh) [inline kronolith/templates/dynamic/edit.inc]: There's a `<p>` tag missing inside the `<div>` tag

File: kronolith/migration/23_kronolith_remote_organizer.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    {
+        $this->addColumn('kronolith_events', 'event_organizer', 'string', array('limit' => 255));
+    }
+
+    /**
+     * Downgrade
+     */
+    public function down()
+    {
+        $this->changeColumn('kronolith_events', 'event_organizer', 'text');
```
Comment (yunosh) [inline kronolith/migration/23_kronolith_remote_organizer.php]: This must be a removeColumn().

File: kronolith/js/kronolith.js [hunk 3]
```diff
@@ -5997,6 +5999,18 @@ KronolithCore = {
         $('kronolithEventLinkExport').up('li').show();
         $('kronolithEventExport').href = Kronolith.conf.URI_EVENT_EXPORT.interpolate({ id: ev.id, calendar: ev.c, type: ev.ty });
 
+        /* Attendance and Status */
+        if (!ev.oy) {
+            $('kronolithEventStatus').disabled = true;
```
Comment (yunosh) [inline kronolith/js/kronolith.js]: Now that I see this in action, it doesn't make any sense to disable the status field. Either I allow attendees to edit their copy of the event or not.

File: kronolith/js/kronolith.js [hunk 4]
```diff
@@ -5997,6 +5999,18 @@ KronolithCore = {
...
+            $('kronolithEventAttend').show();
+        } else {
+            $('kronolithEventStatus').disabled = false;
+            $('kronolithEventAttend').hide();
+        }
+        if (ev.cr) {
+            $('kronolithEventAttendance').setValue(ev.cr);
```
Comment (yunosh) [inline kronolith/js/kronolith.js]: This doesn't seem to work. I didn't check but either chaning this field doesn't save the current reply status (in the attendee's copy), or the drop down isn't populated correctly.

**Comment by yunosh** [comment]
What I'm still misssing from this patch is one of the key points of the original request: the attendees aren't imported if I save a copy of this event.

**Comment by yunosh** [comment]
Ping?

**Comment by ralflang** [comment]
Sorry, I am overloaded... I will most probably assign this task to somebody internally.

**Comment by ralflang** [comment]
See https://github.com/horde/horde/pull/167

**Pull request closed without merging**